### PR TITLE
Update dependencies section: libudev

### PR DESCRIPTION
**Pull request by kubo39**

### CODE REVIEW

File: cargo-espflash/README.md [hunk 1]
```diff
@@ -27,26 +27,24 @@ Supports the **ESP32**, **ESP32-C2/C3/C6**, **ESP32-H2**, **ESP32-P4**, and **ES
 
 If you are installing `cargo-espflash` from source (ie. using `cargo install`) then you must have `rustc>=1.74.0` installed on your system.
 
-If you are running **macOS** or **Linux** then [libuv] must also be installed; this is available via most popular package managers. If you are running **Windows** you can ignore this step.
+If you are running **Linux** then [libudev] must also be installed; this is available via most popular package managers. If you are running **Windows** you can ignore this step.
```
Comment (SergioGasquez) [inline cargo-espflash/README.md:30]: Does macOS already includes `libudev`? If thats the case we should also update the last sentence to include macOs

File: cargo-espflash/README.md [hunk 2]
```diff
@@ -27,26 +27,24 @@ Supports the **ESP32**, **ESP32-C2/C3/C6**, **ESP32-H2**, **ESP32-P4**, and **ES
...
 
-```bash
-# macOS
-brew install libuv
+```console
```
Comment (SergioGasquez) [inline cargo-espflash/README.md:32]: Is there any reason to change this to `console`?

**Review by SergioGasquez — COMMENTED** [review]
Thanks for your contribution! Forgot to update the `libudev` dep here, good catch! Left 2 comments

File: cargo-espflash/README.md (hunk 1 above)
Comment (kubo39) [reply cargo-espflash/README.md:30]: macOS doesn't require lubudev, please see [serialport-rs dependencies](https://github.com/serialport/serialport-rs?tab=readme-ov-file#dependencies).
macOS uses CoreFoundation and IOKit, which is built-in.

File: cargo-espflash/README.md (hunk 2 above)
Comment (kubo39) [reply cargo-espflash/README.md:32]: OK, back to using bash here.

File: cargo-espflash/README.md (hunk 1 above)
Comment (SergioGasquez) [reply cargo-espflash/README.md:30]:
```suggestion
If you are running **Linux** then [libudev] must also be installed; this is available via most popular package managers. If you are running **Windows** or **macOS** you can ignore this step.
```

**Review by SergioGasquez — APPROVED** [review]
LGTM! Thanks again!

**Pull request merged**